src/v2/Apps/Show/Components/ShowViewingRoom.tsx

```diff
@@ -1,22 +1,82 @@
 import React from "react"
-import { Box, BoxProps, ResponsiveBox, Text } from "@artsy/palette"
+import { Box, BoxProps, MediumCard, ResponsiveBox, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowViewingRoom_show } from "v2/__generated__/ShowViewingRoom_show.graphql"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { crop } from "v2/Utils/resizer"
+import { getTagProps } from "v2/Components/ViewingRoomCard"
 
-interface ShowViewingRoom extends BoxProps {}
+interface ShowViewingRoomProps extends BoxProps {
+  show: ShowViewingRoom_show
+}
+
+export const ShowViewingRoom: React.FC<ShowViewingRoomProps> = ({
+  show,
+  ...rest
+}) => {
+  const [{ node: viewingRoom }] = show.viewingRoomsConnection.edges
+
+  const thumbnail = crop(viewingRoom.image.imageURLs.normalized, {
+    width: 900,
+    height: 1200,
+  })
 
-export const ShowViewingRoom = ({ ...rest }) => {
   return (
     <Box {...rest}>
       <Text variant="mediumText" mb={1}>
         Viewing Room
       </Text>
 
-      <ResponsiveBox
-        bg="black10"
-        borderRadius={4}
-        maxWidth="100%"
-        aspectWidth={3}
-        aspectHeight={4}
-      />
+      <ResponsiveBox maxWidth="100%" aspectWidth={3} aspectHeight={4}>
+        <RouterLink to={viewingRoom.href}>
+          <MediumCard
+            width="100%"
+            height="100%"
+            image={thumbnail}
+            title={viewingRoom.title}
+            subtitle={show.partner?.name}
+            tag={getTagProps(
+              viewingRoom.status,
+              viewingRoom.distanceToOpen,
+              viewingRoom.distanceToClose
+            )}
+          />
+        </RouterLink>
+      </ResponsiveBox>
     </Box>
   )
 }
+
+export const ShowViewingRoomFragmentContainer = createFragmentContainer(
+  ShowViewingRoom,
+  {
+    show: graphql`
+      fragment ShowViewingRoom_show on Show {
+        partner {
+          ... on Partner {
+            name
+          }
+          ... on ExternalPartner {
+            name
+          }
+        }
+        viewingRoomsConnection {
+          edges {
+            node {
+              status
+              distanceToOpen(short: true)
+              distanceToClose(short: true)
+              title
+              href
+              image {
+                imageURLs {
+                  normalized
```

Comment:
`normalized`? This is the version that does no resizing at all of the originals (and is used for deep zoom artwork images).

Reply:
AFAIK that's the only image on viewing rooms.

Reply:
ooh this is on a viewing room (sorry, misread)